app/models/legal_aid_application.rb

```diff
@@ -100,6 +102,18 @@ def shared_ownership?
     SHARED_OWNERSHIP_YES_REASONS.include?(shared_ownership)
   end
 
+  def own_home?
+    own_home_mortgage? || own_home_owned_outright?
+  end
+
+  def savings_or_investments?
+    savings_amount.present? && savings_amount.any_value?
+  end
+
+  def other_assets?
+    other_assets_declaration.present? && other_assets_declaration.any_value?
```

Comment:
Would the safe navigation operator work here?
`other_assets_declaration&.any_value?`

Reply:
as regards `__send__` it is now best practice to use this because some libraries override `send` to do something entirely different (e.g. the Socket class uses it to transmit data)

as regards the & operator: I did it like that because I prefer to have methods ending in a ? explicitly return `true` or `false`, rather than nil which is falsey.

Reply:
@stephenrichards thanks for the explanation. It makes sense 👍